src/libmongoc/tests/test-mongoc-client-side-encryption.c

```diff
@@ -6245,12 +6244,29 @@ test_auto_datakeys (void *unused)
 }
 
 static void
-test_create_encrypted_collection_simple (void *unused)
+_do_cec_test (void (*test) (const char *kmsProvider))
+{
+   // Run the test using the "local" key:
+   test ("local");
+   test ("aws");
+}
+
+// Declare a createEncryptedCollection test case (See usage below)
+#define CEC_TEST(name, ...)               \
```

Comment:
I don't have an alternative solution to suggest, but nevertheless wanted to document that this macro pattern gives me mixed feelings. I think I would prefer not using the macro despite the greater verbosity without it, but not strongly enough to suggest that this should be replaced. 😐